docs/docs/guides/smart_contracts/writing_contracts/notes/address_note.md

```diff
@@ -2,13 +2,14 @@
 title: Using Address Note in Aztec.nr
 ---
 
-Address notes hold one unique property - an `address` in the type `AztecAddress`.
+Address notes hold one unique property - an `address` in the type `AztecAddress`. It also holds `npk_m_hash` and `randomness`, similar to other note types.
```

Comment:
"unique" still unclear here. Either it needs clarifying or a different word.
Addresses can be in other notes. The same address can appear in more than one address note?

Zooming out, not sure of the value in telling a dev that an address not has an address in it of type AztecAddress. So consider removing? #LessIsMore